builder/null/builder.go

```diff
@@ -25,7 +25,7 @@ func (b *Builder) Prepare(raws ...interface{}) ([]string, []string, error) {
 		return nil, warnings, errs
 	}
 
-	return nil, warnings, nil
+	return []string{"ID"}, warnings, nil
```

Comment:
HCL2 is strongly typed, meaning we need to know the exact list of variables returned by a builder in order for validation to fail etc. In a following PR I would like to have all builders return exactly what they will share.

Reply:
That also means that a builder that does not have this will not make the variable available.

Reply:
Another solution would be to have a default list

Reply:
I like the default list idea. When you change this I think you will have to update where we write the default placeholder in the generated data list and change how JSON deals with it as well. 
I guess you'll figure out what to do when you do it!